src/main/java/ca/uwaterloo/drinkmasterapi/feature/mqtt/model/PourItem.java

```diff
@@ -0,0 +1,37 @@
+package ca.uwaterloo.drinkmasterapi.feature.mqtt.model;
+
+// not used for mqtt
+public class PourItem {
```

Comment:
rename it to be IngredientItem to be more precise

Reply:
not used, just an example here